rackcity/models/site.py

```diff
@@ -5,3 +5,11 @@ class Site(models.Model):
     abbreviation = models.CharField(max_length=6, unique=True)
     name = models.CharField(max_length=150, unique=True)
     is_storage = models.BooleanField(default=False)
+
+    @staticmethod
```

Comment:
What's this decorator for?

Reply:
if it's a static method, you can access it from the class without needing a reference to an object of the class to call it on (so the same way we're doing `Site.objects.all()` anywhere we want we can also do `Site.get_datacenters()`)